model/src/main/scala/za/co/absa/atum/model/utils/JsonSyntaxExtensions.scala

```diff
@@ -16,16 +16,25 @@
 
 package za.co.absa.atum.model.utils
 
-object OptionImplicits {
-  implicit class OptionEnhancements[T](val option: Option[T]) extends AnyVal {
-    /**
-      * Gets the `option` value or throws the provided exception
-      *
-      * @param exception the exception to throw in case the `option` is None
-      * @return
-      */
-    def getOrThrow(exception: => Throwable): T = {
-      option.getOrElse(throw exception)
+import io.circe.parser.decode
+import io.circe.syntax._
+import io.circe.{Decoder, Encoder}
+
+object JsonSyntaxExtensions {
+
+    implicit class JsonSerializationSyntax[T: Encoder](val obj: T) {
+      def asJsonString: String = obj.asJson.noSpaces
+
+      def asJsonStringPretty: String = obj.asJson.spaces2
```

Comment:
Do we really need this 'pretty' version?

Reply:
It is not used, I guess we don't need it until further notice.